agent/cleanup.sh

```diff
@@ -14,3 +14,8 @@ early_cleanup_validation
 
 rm -rf "${OCP_DIR}/manifests"
 rm -rf "${OCP_DIR}/output"
+
+for host in $(cat "${OCP_DIR}"/hostname); 
```

Comment:
I think the OCP_DIR has already been removed this point https://github.com/openshift-metal3/dev-scripts/blob/master/ocp_cleanup.sh#L19

Instead of using a file, should be able to get the list of hostnames that were added to dns using, for example:
$ sudo virsh net-dumpxml ostestbm | xmllint --xpath "//dns/host/hostname" -

Reply:
Hostnames should be always available from the network definition

Reply:
Save ip and hostname in "ip\<SPACE\>hostname" format
in "${OCP_DIR}"/hosts file. Then, read and parse
this data, use ip just to ssh into the host
and use the hostname in agent gather tar filename.
No need to save into /etc/hosts just to be able to resolve the hostname and also then no need to worry about cleaning.

Reply:
You can remove this file from the PR (agent/cleanup.sh)